runtime/kusama/src/lib.rs

```diff
@@ -1263,7 +1263,7 @@ pub type XcmRouter = (
 
 parameter_types! {
 	pub const Kusama: MultiAssetFilter = Wild(AllOf { fun: WildFungible, id: Concrete(KsmLocation::get()) });
-	pub const KusamaForStatemint: (MultiAssetFilter, MultiLocation) = (Kusama::get(), X1(Parachain(1000)));
+	pub const KusamaForStatemint: (MultiAssetFilter, MultiLocation) = (Kusama::get(), MultiLocation::with_parachain_interior(1000));
```

Comment:
cant you implement `.into()` to make these kinds of transitions easier?

Reply:
Agree. `Parachain(1000).into()` is much more readable.

Reply:
This is unfortunately in a `const` context, and the `into()` function is not `const`. Happy to brainstorm some more ideas to make this more readable however.